core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/jserrorsfilter/JsErrorsFilter.java

```diff
@@ -65,20 +65,34 @@ public void setParameters(Map<String, String> params) throws ParametersException
   @Override
   public Set<JsErrorLog> modifyData(Set<JsErrorLog> data) {
     Set<JsErrorLog> filteredJsErrors = new HashSet<>(data);
+    filteredJsErrors.forEach(jsErrorLog -> {
```

Comment:
Something's wrong with formatting here also.
Btw, you could use streams api here also, sth. like:

```java
filteredJsErrors.stream()
  .filter(this::shouldBeIgnored)
  .forEach(this::addErrorInfo);
```

Of course this approach is also fine.

Reply:
Formatting fixed. I have left the previous non-stream approach to not suggest that we are filtering the errors - we are returning all of them, but some of them modified.